data/www-widgetapi.js

```diff
@@ -0,0 +1,159 @@
+(function(){var g,k=this;function l(a){a=a.split(".");for(var b=k,c;c=a.shift();)if(null!=b[c])b=b[c];else return null;return b}
```

Comment:
I mean, bonus points if we can install this via npm/GitHub or some other managed way so we don't need to host a copy at all, but we can do that in the future.